packages/xo-web/src/xo-app/backup/new/mirror/index.js

```diff
@@ -199,16 +225,55 @@ const NewMirrorBackup = decorate([
       },
       resetMirrorBackup: () => (_, props) => getInitialState(props),
       linkState,
-      toggleMode: (_, { mode }) => ({ mode }),
+      toggleMode: (_, { mode }) => ({ mode, vmsToMirror: undefined }),
```

Comment:
Why are you clearing this when the mode change?

Reply:
If the user select incremental as mode, we will display in the `select` only incremental VM backups.
If the mode changes, the list of selectable VMs changes, so the list of selected VMs may change too

Reply:
That makes sense indeed :+1: